lib/ui/common/components/cobble_fab.dart

```diff
@@ -2,7 +2,7 @@ import 'package:flutter/material.dart';
 
 /// Simple wrapper around Material FAB. Prefer to use this widget instead of
 /// [FloatingActionButton.extended]
-class CobbleFab extends StatelessWidget {
+class CobbleFab extends FloatingActionButton {
```

Comment:
Do you have to inherit it? `CobbleFab` is already using `FloatingActionButton` in its build method, you could pass `onPressed` to  `FloatingActionButton` here: https://github.com/pebble-dev/mobile-app/blob/a65a43f3cdd779aee901d77ec37b6f8d31465aba/lib/ui/common/components/cobble_fab.dart#L29

Reply:
Oh, that wasn't because of that, it was just easier to use it that way with floatingActionButton in CobbleScaffold

Reply:
I don't understand why it was easier, can you help me out, please? What did inheriting from `FloatingActionButton` change?

Reply:
Scaffold requires floatingActionButton to be of type of FloatingActionButton?

Reply:
Cool, thanks for explanation!